react/components/ProductImages/index.js

```diff
@@ -38,8 +38,42 @@ class ProductImages extends Component {
     return null
   }
 
-  static Loader = props =>
-    props.isVertical ? (
+  static Loader = (loaderProps = {}) => {
+    const styles = {
+      'vtex-product-image__selected-image--loader': {
+        x: '10%',
+        y: '5%',
+        'max-width': '19.782em',
```

Comment:
`maxWidth`

Reply:
Also, use the `rem` unit, not `em`

Reply:
Actually, this property doesn't work. It's not necessary.